SQLQuery_2.sql

```diff
@@ -0,0 +1,295 @@
+CREATE DATABASE BillsManagement;
+
+USE BillsManagement;
+
+CREATE TABLE Customers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+FirstName VARCHAR(150),
+LastName VARCHAR(150),
+Email VARCHAR(200),
+);
+
+CREATE TABLE Addresses
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+StreetName VARCHAR(200),
+StreetNumber INT,
+City VARCHAR(150),
+CustomerId INT REFERENCES Customers(Id)
+);
+
+CREATE TABLE Contracts
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+ContractType VARCHAR(150),
+StartDate DATE,
+EndDate DATE,
+);
+
+CREATE TABLE Customers_Contracts
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+CustomerId INT REFERENCES Customers(Id),
+ContractId INT REFERENCES Contracts(Id)
+);
+
+CREATE TABLE Bills
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Amount MONEY,
+Deadline DATE,
+IsPaid BIT,
+ContractId INT REFERENCES Contracts(Id)
+);
+
+INSERT INTO Customers
+VALUES('Ana', 'Mitrea', 'am@gmail.com');
+INSERT INTO Customers
+VALUES('Andrei', 'Stan', 'as@gmail.com');
+INSERT INTO Customers
+VALUES('Cristina', 'Ciocan', 'cc@gmail.com');
+INSERT INTO Customers
+VALUES('Teodor', 'Nistor', 'tn@gmail.com');
+INSERT INTO Customers
+VALUES('Stefania', 'Iancu', 'si@gmail.com');
+
+SELECT *
+FROM Customers;
+
+INSERT INTO Addresses
+VALUES('s1', 10, 'Iasi',1);
+INSERT INTO Addresses
+VALUES('s2', 532, 'Bacau',1);
+INSERT INTO Addresses
+VALUES('s3', 43, 'Ploiesti',2);
+INSERT INTO Addresses
+VALUES('s4', 123, 'Constanta',3);
+INSERT INTO Addresses
+VALUES('s5', 112, 'Suceava',4);
+INSERT INTO Addresses
+VALUES('s6', 132, 'Iasi',5);
+INSERT INTO Addresses
+VALUES('s7', 532, 'Suceava',4);
+INSERT INTO Addresses
+VALUES('s8', 53, 'Cluj',5);
+INSERT INTO Addresses
+VALUES('s9', 75, 'Timisoara',2);
+INSERT INTO Addresses
+VALUES('s10', 43, 'Iasi',2);
+
+SELECT *
+FROM Addresses;
+
+CREATE TABLE ContractTypes
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(25)
+);
+
+ALTER TABLE Contracts
+ADD ContractType INT REFERENCES ContractTypes(Id);
```

Comment:
This script is not running successfully from the start to the end. As an example, because there is no drop command before this add ContractType command you get an error when trying to run it (the column already exists and it either need to be removed before adding it back of it has to be altered instead of added).  See https://github.com/nicoGherasim/wantsomeDotNet7/blob/main/Wantsome.DotNet7/Databases/%5Bweek8%5D%20bills%20management.sql line 91 for the drop command.